mobly/controllers/android_device_lib/services/logcat.py

```diff
@@ -121,8 +120,8 @@ def create_output_excerpts(self, test_info):
         self.OUTPUT_FILE_TYPE, test_info, 'txt'
     )
     excerpt_file_path = os.path.join(dest_path, filename)
-    with io.open(
-        excerpt_file_path, 'w', encoding='utf-8', errors='replace'
+    with open(
+        excerpt_file_path, 'w', encoding='utf-8', errors='replace', newline=''
```

Comment:
add an inline comment explaining that when `newline` is `''`, no conversion happens

Reply:
Done. Added comments for both read and write.